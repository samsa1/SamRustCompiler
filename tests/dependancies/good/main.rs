mod module1;
mod module2;
use module2::Constructor::True;


fn main() {
    module1::exec1();
    module2::exec();
    module2::module3::exec_top();
    let module1 = 3;
    if module1 == 3 {
        print!("test\n")
    }
    let something = True;
    match something {
        True => print!("success\n"),
        module2::Constructor::False => print!("failed\n"),
    }
}