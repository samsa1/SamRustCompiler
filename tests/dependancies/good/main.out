module1
module3
module 3 from far away
